requirements-dev.in

```diff
@@ -39,3 +39,4 @@ pypandoc
 git+https://github.com/someshchaturvedi/customizable-django-profiler.git#customizable-django-profiler
 tabulate==0.8.2
 fonttools
+flower==0.9.4
```

Comment:
I specified this version because when I didn't do it, I was getting this error during compilation:

```Could not find a version that matches prometheus-client==0.7.1,==0.8.0 (from -c requirements.txt (line 66))
Tried: 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10, 0.0.11, 0.0.12, 0.0.13, 0.0.14, 0.0.15, 0.0.16, 0.0.17, 0.0.18, 0.0.19, 0.0.20, 0.0.21, 0.1.0, 0.1.1, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 0.4.1, 0.4.2, 0.5.0, 0.6.0, 0.7.0, 0.7.1, 0.8.0, 0.8.0, 0.9.0, 0.9.0
There are incompatible versions in the resolved dependencies:
  prometheus-client==0.7.1 (from -c requirements.txt (line 66))
  prometheus-client==0.8.0 (from flower==0.9.7->-r requirements-dev.in (line 42))
```

I didn't want to touch production dependencies in this PR. Flower 0.9.4 is the newest version that worked well with our current dependencies. Is this approach okay?

Reply:
Absolutely.